### DISCUSSION TITLE
Issue using Tautulli & Bazarr simultaneously

### DISCUSSION THREAD

**Discussion by ab268-plex** [McCloudS/subgen#134] [Q&A]
I seem to have issues with Bazarr disabling the whisper-asr provider occassionally due to Tautulli already running a transcribe.
Is the idea that we should only use one source, i.e. Bazarr or Tautulli ?

Could always be something else causing bazarr to disable the provider.

**Comment by McCloudS** [comment]
I haven't done any testing on using them both simultaneously, so I can't speculate as to why it may or may not fail.  It would provide a redundant outcome, so I would recommend only using one.  Bazarr is really picky and will disable the provider very quickly for a number of reasons.
